src/api/app/models/user.rb

```diff
@@ -102,6 +102,10 @@ class User < ApplicationRecord
   after_create :create_home_project, :track_create
   before_save :send_metric_for_beta_change, if: :in_beta_changed?
 
+  def self.in_beta?
+    where(in_beta: true)
```

Comment:
Are these lines related to this pull request?

Reply:
not at all :eyes: